DEVELOPER_GUIDE.md

```diff
@@ -55,7 +55,32 @@ Bound addresses will then be logged to the terminal :
 
 ## Run OpenSearch
 
-- Start a seperate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`
+- Start a separate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`.
+
+- Check if extensions directory exists in OpenSearch using `ls`.
+- If the directory does not exist, create it using `mkdir extensions`.
```

Comment:
instead of using relative paths, consider adding use `{projectRoot}` or another shorthand to make it easier to understand where this directory should be created

Reply:
I'm of the opposite view; I expect people will be copy/pasting the commands from here and I think a placeholder like that would confuse people. Unless we want them to type a command which sets an environment variable and then use that?  Which seems overkill.

Reply:
A place holder is one way to help the users of the guide orient themselves, if you mentioned "navigate to the root of the OpenSearch git repository" or "navigate to the OpenSearch installation directory" that would also provide the context to help the use know they are following the guide correctly.